src/Nancy/Helpers/HttpEncoder.cs

```diff
@@ -1,7 +1,7 @@
 //
 // Authors:
 //   Patrik Torstensson (Patrik.Torstensson@labs2.com)
-//   Wictor Wil�n (decode/encode functions) (wictor@ibizkit.se)
+//   Wictor Wil�n (decode/encode functions) (wictor@ibizkit.se)

```

Comment:
i spent 1 morning with vs2015, and it did crap like this, im going back to atom.